src/item.cpp

```diff
@@ -2094,9 +2096,10 @@ std::string item::tname( unsigned int quantity, bool with_prefix ) const
                 if (damage == 4) damtext = rm_prefix(_("<dam_adj>pulped "));
 
             } else if ( OPTIONS["ITEM_HEALTH_BAR"] ) {
-                auto const &nc_text = get_item_hp_bar(damage);
-                damtext = "<color_" + string_from_color(nc_text.second) + ">" + nc_text.first + " </color>";
-
+                if(!grid) {
```

Comment:
Out of curiosity: why is the item health bar not added here when in grid view? The code in inventory_ui.cpp seems to add it anyway. This adds quite some dependency between this function and the inventory code.